### ISSUE TITLE
All fetched datatypes should use backpressure

### ISSUE THREAD

**Issue by cspramit** [casper-network/casper-node#3028]
Adding additional channels, we can reshape the way requests for blocks and deploys are made in the same way we did it for trie nodes. As a result, we can stop tweaking request rates for blocks manually in the fast sync code.

**Comment by marc-casperlabs** [comment]
@piotr-dziubecki This is postponed for two sprints, waiting on 1.5 changes to be merged into `dev` and into a release branch thereafter, after @Fraser999 has voiced (justified) concerns of keeping the changeset for 1.5 as small as possible.
